sdlib/d/gc/emap.d

```diff
@@ -27,9 +27,12 @@ public:
 		return leaf is null ? PageDescriptor(0) : leaf.load();
 	}
 
+	bool map(void* address, size_t size, PageDescriptor pd) shared {
```

Comment:
I'm nto quite sure why the other function is called remap, but why is this one map?